publish-maven.gradle

```diff
@@ -1,102 +1,87 @@
-			}
-			licenses {
-				license {
-					name 'The Apache Software License, Version 2.0'
-					url 'https://www.apache.org/licenses/LICENSE-2.0.txt'
-					distribution 'repo'
+publishing {
+	publications {
+		mavenJava(MavenPublication) {
+			from components.java
+			pom {
+				afterEvaluate {
+					name = project.description
+					description = project.description
 				}
-			}
-			scm {
-				url = linkScmUrl
-				connection = 'scm:git:' + linkScmConnection
-				developerConnection = 'scm:git:' + linkScmDevConnection
-			}
-			developers {
-				developer {
-					id = 'garyrussell'
-					name = 'Gary Russell'
-					email = 'grussell@pivotal.io'
-					roles = ['project lead']
+				url = 'https://github.com/spring-projects/spring-amqp'
+				organization {
+					name = 'Spring IO'
+					url = 'https://spring.io/projects/spring-ampq'
 				}
-				developer {
-					id = 'artembilan'
-					name = 'Artem Bilan'
-					email = 'abilan@pivotal.io'
+				licenses {
+					license {
+						name = 'Apache License, Version 2.0'
+						url = 'https://www.apache.org/licenses/LICENSE-2.0'
+						distribution = 'repo'
+					}
 				}
-				developer {
-					id = 'dsyer'
-					name = 'Dave Syer'
-					email = 'dsyer@pivotal.io'
+				scm {
+					url = linkScmUrl
+					connection = "scm:git:$linkScmConnection"
+					developerConnection = "scm:git:$linkScmDevConnection"
 				}
-				developer {
-					id = 'markfisher'
-					name = 'Mark Fisher'
-					email = 'mfisher@pivotal.io'
+				developers {
+					developer {
+						id = 'garyrussell'
+						name = 'Gary Russell'
+						email = 'grussell@pivotal.io'
+						roles = ['project lead']
+					}
+					developer {
+						id = 'artembilan'
+						name = 'Artem Bilan'
+						email = 'abilan@pivotal.io'
+					}
+					developer {
+						id = 'dsyer'
+						name = 'Dave Syer'
+						email = 'dsyer@pivotal.io'
+					}
+					developer {
+						id = 'markfisher'
+						name = 'Mark Fisher'
+						email = 'mfisher@pivotal.io'
+					}
+					developer {
+						id = 'mpollack'
+						name = 'Mark Pollack'
+						email = 'mpollack@pivotal.io'
+					}
+					developer {
+						id = 'jbrisbin'
+						name = 'Jon Brisbin'
+						email = 'jbrisbin@pivotal.io'
+					}
+					developer {
+						id = 'tomas.lukosius'
+						name = 'Tomas Lukosius'
+						email = 'tomas.lukosius@opencredo.com'
+					}
 				}
-				developer {
-					id = 'mpollack'
-					name = 'Mark Pollack'
-					email = 'mpollack@pivotal.io'
+				issueManagement {
+					system = 'GitHub'
+					url = 'https://github.com/spring-projects/spring-amqp/issues'
 				}
-				developer {
-					id = 'jbrisbin'
-					name = 'Jon Brisbin'
-					email = 'jbrisbin@pivotal.io'
+			}
+			versionMapping {
+				usage('java-api') {
+					fromResolutionResult()
 				}
-				developer {
-					id = 'tomas.lukosius'
-					name = 'Tomas Lukosius'
-					email = 'tomas.lukosius@opencredo.com'
+				usage('java-runtime') {
+					fromResolutionResult()
 				}
 			}
 		}
 	}
 }
+
+//artifactoryPublish {
```

Comment:
We need to check this from CI plan.
It looks like SF uses this because they don't have this section:
```
artifacts {
	archives sourcesJar
	archives javadocJar
}
```

So, we need to keep an eye on this if we get some failures on Bamboo.